evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -188,6 +188,9 @@
             <a href="{% url 'staff:evaluation_copy' evaluation.id %}" class="btn btn-sm btn-light" data-bs-toggle="tooltip" title="{% translate "Copy" %}">
                 <span class="fas fa-copy"></span>
             </a>
+            {% if not evaluation.has_exam %}
+            <button id="openModalBtn" type="button" class="btn btn-sm btn-light" evaluation_id="{{ evaluation.id }}"><span class="fas fa-file-pen fa-fw"></span></button>
```

Comment:
What is the difference to the `show-button` in the form. Is this the right way to use the new modals @niklasmohrin ?

Reply:
No, the `confirmation-modal` element should be placed wherever the button to open that modal is. Then, only the `slot=show-button` element will be shown on the normal page and clicking it automatically triggers the modal to open. You can see this in the other usages throughout the codebase

Reply:
If the button to trigger the modal is far away from the form it triggers, you can use `form=the-id-of-the-other-form` on `confirmation-modal` to link it to the correct form